utils.py

```diff
@@ -475,6 +477,45 @@ def get_human_readable_time_string(time_msec):
         '%B %d %H:%M:%S', time.gmtime(python_utils.divide(time_msec, 1000.0)))
 
 
+def create_string_from_largest_unit_in_timedelta(timedelta_obj):
+    """Given the timedelta object, find the largest non zero time unit and
+    return that value, along with the time unit, as a human readable string.
+    The returned string is not localized.
+
+    Args:
+        timedelta_obj: datetime.timedelta. A datetime timedelta object. Datetime
+            timedelta objects are created when you subtract two datetime
+            objects.
+
+    Returns:
+        str. A human readable string representing the value of largest non zero
```

Comment:
largest --> the largest

non zero --> non-zero or nonzero

Reply:
Done